hutch_python/epics_arch.py

```diff
@@ -28,19 +28,25 @@ def _create_parser():
 
     parser.add_argument('--dry-run', action='store_true', default=False,
                         help='Print to stdout what would be written in the '
-                        'archFIle.')
+                        'archFile.')
+
+    parser.add_argument('--update', action='store_true', default=False,
```

Comment:
I dont think we need that argument, it serves the same purpose as --dry-run (in fact it does the exact opposite).

Reply:
Sounds good! I am working on this now